shinydashboard/lantern/functions/endpoints.R

```diff
@@ -465,6 +465,27 @@ get_cap_stat_sizes <- function(db_connection) {
     tidyr::replace_na(list(fhir_version = "Unknown"))
 }
 
+get_validation_results <- function(db_connection) {
+  res <- tbl(db_connection,
```

Comment:
Just thinking about the long term when this gets integrated with the version feature stuff, we'd want all URL/FHIR version pairs represented on this page, not just one entry per URL, right? @mayerm94

Reply:
Is this something I should be able to include yet or not since the validation feature branch does not have the version information? Should it be left as is for now or is there something I should do now in order to have it prepared for the version feature stuff?

Reply:
I guess I was just more noting that we'll have to come back to this! I just added a note to the "cleanup versions" task as a reminder to look through all of the shiny dashboard files and figure out which visualizations might need to be updated!